fix/gsi/anavinfo.rrfs_all

```diff
@@ -0,0 +1,81 @@
+met_guess::
```

Comment:
About the name of this file: should have a name fit to the application. Using "_all" is too general.